emgD.C

```diff
@@ -101,232 +114,258 @@ class ExRootTreeReader;
+			"number of tracks", "number of events",
+			500, 0.0, 500.0);
 
+	plots->ftrkPT = result->AddHist1D(
+			"track_pt", "track P_{T}",
+			"track P_{T}, GeV/c", "number of tracks",
+			50, 0.0, 50.0);
 
-  // book histograms for jets
 
+	plots->ftrkTH = result->AddHist1D(
+			"track_th", "track TH2D",
+			"track TH2D", "number of tracks",
+			50, 0.0, 0.7);
 
-  plots->fnJet = result->AddHist1D(
-    "nJet", "number of jets",
-    "number of jets", "number of events",
-    50, 0.0, 50.0);
+	plots->ftrkD0 = result->AddHist1D(
+			"track_d0", "track D_{0}",
+			"track D_{0}, mm", "number of tracks",
+			50, -1.0, 1.0);
 
-  plots->fJetPT = result->AddHist1D(
-    "jet_pt", "jet P_{T}",
-    "jet P_{T}, GeV/c", "number of jet",
-    50, 0.0, 500.0);
+	plots->ftrkD0Error = result->AddHist1D(
+			"track_d0Error", "track D_{0} Error",
+			"track D_{0} Error, mm", "number of tracks",
+			50, -0.5, 0.5);
 
 
-  plots->fJetAM = result->AddHist1D(
-    "jet_alphamax", "jet alphamax",
-    "alphamax 4 leading jets", "number of jet",
-    50, 0.0, 1);
 
+	plots->ftrkD0sig = result->AddHist1D(
+			"track_d0sig", "track D_{0} sig",
+			"track D_{0} sig", "number of tracks",
+			50, -5.0, 5.0);
 
-  plots->fJetAMp = result->AddHist1D(
-    "jet_alphamaxp", "jet alphamaxp",
-    "alphamaxp 4 leading jets", "number of jet",
-    50, 0.0, 1);
 
+	// book histograms for jets
 
-  plots->fJetD0max = result->AddHist1D(
-    "jet_D0max", "jet d0 max",
-    "d0max 4 leading jets", "number of jet",
-    50, 0.0, 1.0);
 
-  plots->fJetD0med = result->AddHist1D(
-    "jet_D0med", "jet d0 med",
-    "d0med 4 leading jets", "number of jet",
-    50, 0.0, 1.0);
+	plots->fnJet = result->AddHist1D(
+			"nJet", "number of jets",
+			"number of jets", "number of events",
+			50, 0.0, 50.0);
 
-  plots->fJetTHmed = result->AddHist1D(
-    "jet_THmed", "jet th med",
-    "theta2d med 4 leading jets", "number of jet",
-    50, 0.0, 0.7);
+	plots->fJetPT = result->AddHist1D(
+			"jet_pt", "jet P_{T}",
+			"jet P_{T}, GeV/c", "number of jet",
+			50, 0.0, 750.0);
 
-  plots->fdqd0 = result->AddHist1D(
-    "fdqd0", "dq d0",
-    "impact parameter for tracks in jets matched to dark quarks", "number of tracks",
-    50, -1., 1.);
 
+	plots->fJetAM = result->AddHist1D(
+			"jet_alphamax", "jet alphamax",
+			"alphamax 4 leading jets", "number of jet",
+			50, 0.0, 1.0);
 
-  plots->fdd0 = result->AddHist1D(
-    "fdd0", "d d0",
-    "impact parameter for tracks in jets matched to down quarks", "number of tracks",
-    50, -1., 1.);
 
-  plots->fnBJet = result->AddHist1D(
-    "nBJet", "number of bjets",
-    "number of bjets", "number of events",
-    50, 0.0, 50.0);
-  
-  plots->fBJetPT = result->AddHist1D(
-    "bjet_pt", "bjet P_{T}",
-    "bjet P_{T}, GeV/c", "number of bjets",
-    50, 0.0, 500.0);
+	plots->fJetAMp = result->AddHist1D(
+			"jet_alphamaxp", "jet alphamaxp",
+			"alphamaxp 4 leading jets", "number of jet",
+			50, 0.0, 1.0);
 
-  // book histograms for fat jets
```

Comment:
You have somehow deleted all the updates I made to add jet substructure histograms. Please remake your changes so they don't delete other updates.